index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/d86848cfe0.js"></script>
+    <title>Sign Up</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header>
+        <div class="header-section">
+            <p class="normal_desc">Already have an intuit account?</p>
+            <a href="#" class="normal_link">I forgot my user iD or password</a>
+        </div>
+        <a class="button" href="#"><i class="fa fa-lock" aria-hidden="true"></i> Sign in</a>
+    </header>
+    <main>
+      <div class="intuit_logo">
+          <img src="images/intuit_logo.png" alt="intuit logo" width="130">
+      </div>
+
+      <div class="icons">
+        <div class="icon">
+          <img src="images/icon1.png" alt="mint-logo" width="25" />
+          <p>mint</p>
+        </div>
+        <div class="icon">
+          <img src="images/icon2.svg" alt="mint-logo" width="25" />
+          <p>quickbooks</p>
+        </div>
+        <div class="icon">
+          <img src="images/icon3.png" alt="mint-logo" width="25"/>
+          <p>turbotax</p>
+        </div>
+      </div>
+
+      <form action="index.html" method="post">
+        <h1 class="center">Create an Intuit account</h1>
+        <p class="large_desc center">One account for everything Intuit, including Mint.</p>
+        <a href="#" class="normal_link center">Learn more</a>
+
+        <label for="email">Email address</label>
+        <input type="text" id="email" name="email" value="">
+
+        <label for="phone">Phone (recommended)</label>
+        <input type="text" id="phone" name="email" value="">
+        <p class="small_desc">Standard call, messaging or data rates may apply.</p>
+
+        <label for="password">Password</label>
+        <input type="password" id="password" name="email" value="">
+
+        <label for="conf_password">Confirm Password</label>
+        <input type="password" id="conf_password" name="email" value="">
+
+        <a class="submit_button" href="#"><i class="fa fa-lock" aria-hidden="true"> </i>Create Account</a>
```

Comment:
Your form should have button or an input tag type submit.

It has a `<button>` or an `<input>` of type `submit` that matches the style of the original page (Blue background, white text) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/html_forms/form_and_input_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59524927/))

Reply:
Thank you for your feedback,
do I have to make a new pull request, after I made modifications?